package.json

```diff
@@ -44,12 +44,13 @@
   },
   "devDependencies": {
     "@prettier/plugin-xml": "^3.4.1",
-    "@types/multer": "^1.4.12",
-    "@types/jest": "^29.5.14",
```

Comment:
I have an inkling that the removal of these two dependencies will cause some issues with the pre-commit hooks in some circumstances.... or at least that is what my memory is telling me

Reply:
Yeah.. I was surprised by the specificness of it... But couldn't get it to fail easily...

Reply:
But why are these dependencies there anyway... 🫤

Reply:
Ok, if you tried then I say let's get rid of them and hope for the best 😄